File: packages/gunreip/laravel-translation-workbench/resources/views/pages/tw-graph/samples/documentation/idea-to-paper/flow/while/code-examples/while-nested.js
const groups = loadGroups();
let groupIndex = 0;
while (groupIndex < groups.length) {
    const items = groups[groupIndex];
    let itemIndex = 0;
    while (itemIndex < items.length) {
        processItem(items[itemIndex]);
        itemIndex = itemIndex + 1;
    }
    groupIndex = groupIndex + 1;
}
showSummary();

File: packages/gunreip/laravel-translation-workbench/resources/views/pages/tw-graph/samples/documentation/idea-to-paper/flow/while/code-examples/while-nested.cpp
auto groups = loadGroups(); // std::vector<std::vector<Item>>
std::size_t groupIndex = 0;
while (groupIndex < groups.size()) {
    const auto& items = groups[groupIndex];
    std::size_t itemIndex = 0;
    while (itemIndex < items.size()) {
        processItem(items[itemIndex]);
        itemIndex = itemIndex + 1;
    }
    groupIndex = groupIndex + 1;
}
showSummary();
